xml/installation-installation-ses_integration.xml

```diff
@@ -380,4 +380,49 @@ rgw keystone verify ssl = false # If keystone is using self-signed
    </step>
   </procedure>
  </section>
+ <section>
+   <title>Enable Copy-On-Write Cloning of Images</title>
+   <para>
+    Due to a security issue described in <link
+    xlink:href="http://docs.ceph.com/docs/master/rbd/rbd-openstack/?highlight=uuid#enable-copy-on-write-cloning-of-images"
+    />, the copy-on-write cloning of images is not supported when
```

Comment:
Do we say "not supported" or not recommended?

Reply:
This is literally a backport of the original, it was reviewed by @bryanstephenson and deemed okay language. Something being not support and something being not recommended are very different. Clarification is important :)